eth/downloader/downloader.go

```diff
@@ -1363,7 +1360,7 @@ func (d *Downloader) fetchReceipts(from uint64, beaconMode bool) error {
 // processHeaders takes batches of retrieved headers from an input channel and
 // keeps processing and scheduling them into the header chain and downloader's
 // queue until the stream ends or a failure occurs.
-func (d *Downloader) processHeaders(origin uint64, ttd *big.Int, beaconMode bool) error {
```

Comment:
You mean just the new unused `td` variable in the signature? IMO it's ok to have that, since the linter doesn't complain about it and apparently it'll make merge conflicts one line fewer. The rest of the Zurich logic looks in place to me.

Re: Zurich; See also `merge/foundation-release/1.13.5` (https://github.com/etclabscore/core-geth/pull/593), in case they or I fixed something in this version in the later one.